src/microbit/pxt_hermegaasa/hermegaasa.md

```diff
@@ -0,0 +1,249 @@
+level: 2
+author: Felix Bjerke og Tjerand Silde
+language: nb
+---
+
+# Introduksjon {.intro}
+
+Hermegåsa er et spill der en person er *spilleder*, og går ut på at han utfører instruksjoner på micro:biten sin som de andre spillerene skal gjenta, altså herme etter, på sin micro:bit. Dette er et multiplayer-spill, og det er om å gjøre å være den raskeste spilleren til å herme etter spillederen. Spillederen har oversikt på skjermen sin over poengene til de ulike spillerene, og spillet avsluttes når en spiller har vunnet 5 ganger.
+
+![](hermegaasa.png)
+
+På bildet ser du at spillederen er i midten og har poengsummen til spiller 1 og spiller 2 på skjermen sin. På sidene har spillerne fått beskjed om hva de skal gjøre, i form av en pil på skjermen. Førstemann som vipper micro:biten sin sammen vei som pilen vinner.
+
+# Steg 1: Sjekk at du har riktig utstyr {.activity}
+
+  *Det er viktig at du har alt utstyr og tilbehør for å kunne gjøre denne oppgaven.*
+
+  ## Sjekkliste {.check}
+
+  - [ ] 1 micro:bit med en micro-usb-kabel som spilleder.
+
+  - [ ] 2-5 micro:biter med strømforsyning (micro-usb-kabler eller batterier).
+
+  - [ ] En datamaskin med Internett.
+
+### Spillregler {.protip}
+
+  *Her er en oversikt over reglene i spillet:*
+
+   Skjerm viser          | Spiller utfører             |
+  ---------------------- | --------------------------- |
+       A                 | Trykk på A                  |
+       B                 | Trykk på B                  |
+       C                 | Trykk på A + B              |
+       <                 | Vipp til venstre            |
+       \>                | Vipp til høyre              |
+
+# Steg 2: Programmere spillederen {.activity}
+
+*Når spillet skal starte så må vi sørge for at alle micro:bitene kan kommunisere med hverandre, og da må vi bestemme et gruppe-nummer som alle deltagere bruker når man sender informasjon over Bluetooth. I tilleg må vi lagre at vi har sendt ut en melding til deltagere, slik at vi ikke sender ut flere meldinger før vi har fått svar på den forrige.*
+
+## Sjekkliste {.check}
+
+- [ ] Start et nytt PXT-prosjekt, for eksempel ved å gå til
+  [makecode.microbit.org](https://makecode.microbit.org/?lang=no).
+
+- [ ] Finn blokken `ved start` under *Basis* og dra den inn i kodefeltet.
+
+- [ ] Gå til *Radio* og finn blokken `radio sett gruppe`. I utgangspunktet står det tallet `1` i denne blokken, men du bestemmer selv hvilket tall du vil bruke.
+
+- [ ] Til slutt går vi til *Variabler* og velger blokken `sett variabel til`. Denne kan du bytte navn på ved å klikke på _variabel_ og så på _Rename variable_. Den bør hete _MessageSent_.
+
+- [ ] Til _MessageSent_ kobler vi blokken `usann`, som du finner under *Logikk*.
+
+- [ ] Når spillet starter skal alle spillerne ha `0` poeng. Lag en variabel for hver spiller med navn `Player1`, `Player2` osv. som du setter til `0` poeng.
+
+- [ ] Dersom du har gjort alt rett så vil koden din se slik ut:
+
+	![](start.png)
```

Comment:
legger pausen mellom `hvis-blokkene` istedenfor, så sjekkes det ihvertfall for at man venter på et svar, slik at man ikke kan få spillet til å pause før man har fått noe man skal herme etter.  pr nå skal klientene kun få lov til å svare dersom de har svart korrekt, så da vil spillet kun pause når førstemann har svart korrekt.

ja, eg er enig med deg i at serveren burde ha logikk for dette istedenfor at det ligger hos klienten. eg vil oppfordre de til å jukse og endre dette i slutten av oppgaven, og legger til flere punkter enn bare dette ene om dette i slutten av oppgaven.